plugins/io.sarl.lang/src/io/sarl/lang/jvmmodel/SARLJvmModelInferrer.java

```diff
@@ -2220,7 +2220,7 @@ protected void translateSarlFormalParameters(
 			List<? extends XtendParameter> params,
 			final boolean isForInterface,
 			List<InferredStandardParameter> paramSpec) {
-		JvmFormalParameter lastParam = null;
+		final JvmFormalParameter lastParam = owner.getParameters().get(owner.getParameters().size() - 1);
```

Comment:
`owner.getParameters().size() - 1` may produce index-out-of-bounds exception at this location.